Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.opensource.zalan.do/stups/openjdk:latest
```

Comment:
I assume that `pierone.stups.zalan.do/pitchfork/java-8-ubuntu-20.04` contains the instructions that install AWS RDS CA bundle, right ?

Imho, I think it would be nice to use a centralized base image (e.g. `registry.opensource.zalan.do/library/openjdk-8:latest`) and we explicitly specify the certificate import instructions in the following Dockerfile for clarity. wdyt @GoWind ?

```dockerfile
FROM registry.opensource.zalan.do/library/openjdk-8:latest

MAINTAINER Zalando SE

# add AWS RDS CA bundle
RUN mkdir /tmp/rds-ca && \
    curl https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem > /tmp/rds-ca/aws-rds-ca-bundle.pem
# split the bundle into individual certs (prefixed with xx)
# see http://blog.swwomm.com/2015/02/importing-new-rds-ca-certificate-into.html
RUN cd /tmp/rds-ca && csplit -sz aws-rds-ca-bundle.pem '/-BEGIN CERTIFICATE-/' '{*}'
RUN for CERT in /tmp/rds-ca/xx*; do mv $CERT /usr/local/share/ca-certificates/aws-rds-ca-$(basename $CERT).crt; done
RUN update-ca-certificates

COPY resources/api/kio-api.yaml /zalando-apis/
COPY target/kio.jar /
COPY java-dynamic-memory-opts /bin/java-dynamic-memory-opts

EXPOSE 8080
ENV HTTP_PORT=8080

CMD java $JAVA_OPTS $(java-dynamic-memory-opts 70) -jar /kio.jar
```

Reply:
This sounds like the ideal solution. We can start off by basing the changes we need in the Dockerfile on the approved image.  I will create a Pull Request with these changes in later. Thanks for taking a look into this. Much appreciated !